packages/compass-e2e-tests/tests/connection.test.ts

```diff
@@ -425,7 +460,12 @@ describe('Connection screen', function () {
 
     const connectionString = parsedString.toString();
 
+    await browser.setFeature('enableShell', true);
     await browser.connectWithConnectionString(connectionString);
+
+    const shellSection = await browser.$(Selectors.ShellSection);
+    await shellSection.waitForDisplayed();
```

Comment:
Do you know why we need these changes? `enableShell` should default to `true` when Compass is started normally, right?

Reply:
I think I made this change while was debugging particular test suits and this was required in case I run `describe.only()`, because initially, settings apply only when all tests are run.

Reply:
Yeah, it is still the case. if I run `describe.only('Connection screen', function () {})` the shell is hidden by default.

<img width="1432" alt="Screenshot 2022-11-10 at 13 14 04" src="https://user-images.githubusercontent.com/16307679/201089047-ffe5fd07-37c8-4bb4-9205-0691fa18edeb.png">

Reply:
Yeah, it’s this: https://github.com/mongodb-js/compass/blob/a83dab45ca4da6c7597406d8d02e1133fa312044/packages/compass-e2e-tests/helpers/compass.ts#L405-L408

We should definitely not be doing that.